dynamodb-streams-lambda-filters-tf/README.md

```diff
@@ -0,0 +1,80 @@
+# DynamoDB Stream to AWS Lambda with Filters 
+
+This pattern demonstrates how to lambda functions that can recieve DynamoDB Stream with filters. The IaC has been definied in Terraform. 
+
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
```

Comment:
The url is https://serverlessland.com/patterns/dynamodb-streams-lambda-filters-tf